android/beagle/src/main/java/br/com/zup/beagle/newanalytics/ActionAnalyticsConfig.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.newanalytics
+
+import br.com.zup.beagle.core.BeagleJson
+
+/**
+ * This class is used to pass some properties for analytics
+ * @param attributes by default is null, but can be a List of String to indicate what attributes of the action will be
+ * reported. When this param is null, the analytics will get the attribute from AnalyticsConfig.
+ * @param additionalEntries by default is null, but can be a Map of String to Any, on this param can be passed
+ * additional values to be reported with the action.
+ */
+@BeagleJson
+data class ActionAnalyticsProperties(
```

Comment:
Consider the suggestion only if changing the name of the action.
```suggestion
data class AnalyticsActionProperties(
```

Reply:
Here is the same idea on the ActionAnalyticsConfig